scripts/render-notebooks.R

```diff
@@ -113,6 +113,7 @@ rmarkdown::render(tmp_file,
     toc = TRUE, toc_depth = 2,
     toc_float = TRUE, number_sections = TRUE,
     df_print = "paged"
+    css = file.path("..", "components", "styles.css")
```

Comment:
I don't like this precarious way I've designated this file path but `html_document()` wants it relative to the Rmd and I'm not sure how to make this less precarious.

Reply:
I would assume a full path will always work? If so: 
```suggestion
    css = normalizePath(file.path("components", "styles.css"))
```
As you do this, it may make sense to move `references.bib` into `components` as well.